free-1.md

```diff
@@ -4,51 +4,99 @@
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
+A language for describing bridge hands in a manner that is easy to process by a
+machine and by humans.  This would help people right down and save hands for
+future study.  They could perform various operatinons on the hands, such as
+searching for particular contracts, results, and other such metrics.
 
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+I'm helping bridge players.  This would help bridge players who want to study
+hands, share hands with other players, and just keep a database of hands.
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A DSL would be nice especially for making it easy to perform operations on the
+hands.  For example if you just write a hand down on pen and paper it requires
+work to search through the papers and find the hands.  Also you can't analyze
+what the best result would've been if you played it optimally (not that there
+are a few ways to define optimally here...).
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I love to play bridge and often come across interesting hands that I play very
+well and very poorly.  It would be nice to have an easy way to share these hands
+with my friends.
 
 ### Domain
 _Describe the project's domain in five words._
 
+Bridge Players, Authors, and Enthusiasts
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+I think it should look like an easy to read text file with some particular
+format.  Perhaps put the hand first, then the auction, then the order of the
+cards played, then the result.
```

Comment:
It's a little hard for non-experts to understand this part of your description. Some more background on the domain would help, e.g., by defining what hands and auctions are and by describing some of the things that bridge players think about when they're evaluating a hand.